.idea/IB-Trading-Models-And-Backtester.iml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file was probably added before .gitignore was modified. .idea folder should be removed from the repository.